.vscode/settings.json

```diff
@@ -22,5 +22,8 @@
   "javascript.preferences.importModuleSpecifier": "auto",
   "typescript.preferences.importModuleSpecifier": "auto",
   "javascript.preferences.quoteStyle": "single",
-  "typescript.preferences.quoteStyle": "single"
+  "typescript.preferences.quoteStyle": "single",
+
+  "editor.formatOnSave": true,
```

Comment:
We can't turn this on at a project level.
I tried this before, but it causes issues if people don't have the prettier extension installed, or have it installed along with another formatter (such as beautify)

Reply:
for reference, me turning it on:
https://github.com/bradzacher/eslint-plugin-typescript/pull/182
And off again:
https://github.com/bradzacher/eslint-plugin-typescript/pull/192

Reply:
Is there a way to enable auto-format on save without `.vscode/settings.json`?

Always I use `vscode-eslint` to format code, so I have disabled embedded format at all, to avoid conflicts. I can enable/disable specific VSCode extensions for only this repo, but can I change settings without `.vscode/settings.json`?

Reply:
i don't think you can, unfortunately.

I'm hoping that vscode adds committable project-level extension disabling (see https://github.com/Microsoft/vscode/issues/18386#issuecomment-463057761)

Reply:
:sob: